docs/storage/disks_and_volumes.md

```diff
@@ -2131,7 +2131,7 @@ The following strategies are supported:
   * `Replacement`: the update volumes will be replaced upon the VM restart.
   * `Migration`: the update of the volumes will trigger a storage migration of
     the old volumes to the new ones. More details about volume migration can be
-    found in the [volume migration documentation](docs/storage/volume_migration.md).
+    found in the [volume migration documentation](https://kubevirt.io/user-guide/storage/volume_migration/).
```

Comment:
I'd rather prefer a relative link here
```suggestion
    found in the [volume migration documentation](/storage/volume_migration/).
```

Note: the resulting link from the preview looks like this

https://deploy-preview-821--kubevirt-user-guide.netlify.app/storage/volume_migration/

Reply:
Done, thanks!